install/kubernetes/cilium/files/cilium-envoy/configmap/bootstrap-config.json

```diff
@@ -60,6 +60,61 @@
         ]
       },
       {{- end }}
+      {{- if and .Values.envoy.debug.admin.enabled }}
+      {
+        "name": "envoy-admin-listener",
+        "address": {
+          "socket_address": {
+            "address": "0.0.0.0",
```

Comment:
would it be enough to bind the listener only on the loopback interface(s). In combination with `kubectl port-forward` a user could start the debug session on the admin interface. This way it's even more obvious that this listener should only used for debug purposes.

Reply:
@bdecoste what do you think of this proposal?